web-marketplace/src/pages/BasicInfo/BasicInfo.tsx

```diff
@@ -20,13 +21,16 @@ const BasicInfo: React.FC<React.PropsWithChildren<unknown>> = () => {
       onChainProject,
     });
 
-  let initialFieldValues: BasicInfoFormValues | undefined;
-  if (metadata) {
-    initialFieldValues = {
-      'schema:name': metadata['schema:name'],
-      'regen:projectSize': metadata['regen:projectSize'],
-    };
-  }
+  const initialValues: BasicInfoFormSchemaType = useMemo(
+    () => ({
+      'schema:name': metadata?.['schema:name'] ?? '',
+      'regen:projectSize': metadata?.['regen:projectSize'] ?? {
+        'qudt:numericValue': 0,
```

Comment:
this means 0 is displayed as default project size if this hasn't been filled yet, is that ok @erikalogie ?

![image](https://github.com/regen-network/regen-web/assets/3149358/df1a480c-aa39-45bf-95b9-babdbcafa3ad)

Also it seems like there's too much space between the label and the input below.

Reply:
Will fix that.